anet.yml

```diff
@@ -152,8 +152,12 @@ dictionary:
   ADMINISTRATOR_POSITION_TYPE_TITLE: ANET Administrator
   PRINCIPAL_ORG_NAME: Afghan Government Organization
   ADVISOR_ORG_NAME: Advisor Organization
-  POAM_LONG_NAME: Plan of Action and Milestones / Pillars
-  POAM_SHORT_NAME: PoAM
+  TASK_LONG_NAME: Plan of Action and Milestones / Pillars
+  TASK_SHORT_NAME: Task
+  TASK_PROJECTED_COMPLETION: 'Fill in the projected completion date'
```

Comment:
It seems to me that this shoube called TASK_PROJECTED_COMPLETION_PLACEHOLDER, and there should also be a TASK_PROJECTED_COMPLETION_LABEL, in accordance to whatever format is adopted in GH-311